Corona-Warn-App/src/main/res/layout/home_submission_status_card_unregistered.xml

```diff
@@ -39,7 +39,7 @@
 
         <ImageView
             android:id="@+id/icon"
-            android:layout_width="wrap_content"
+            android:layout_width="95dp"
```

Comment:
why did you change this?

Reply:
Oh that was a unintended change. Just reverted it. Thanks for mentioning 👍🏼